mlrun/config.py

```diff
@@ -244,6 +244,14 @@
             # None will set this to be equal to the httpdb.max_workers
             "connections_pool_size": None,
             "connections_pool_max_overflow": None,
+            # below is a db-specific configuration
+            "mysql": {
+                # comma separated mysql modes (globally) to set on runtime
+                # optional values (as per https://dev.mysql.com/doc/refman/8.0/en/sql-mode.html#sql-mode-full):
+                #
+                # if not set, nothing would be set
+                "modes": "STRICT_TRANS_TABLES",
```

Comment:
I don't think you would be able to override it using environment variables if it isn't empty by default, as it uses an `OR` logic to resolve the mode.
Possible solution would be to have `mode: <enabled/disabled>` , and another attribute for a list of modes, that way you would be able to manipulate the modes without the need to change the code

Reply:
I decided that values such as `nil`/`none` will be translated to "none" so that when setting `"modes":"nil"` - nothing would be applied